authors/apps/articles/migrations/0001_initial.py

```diff
@@ -1,5 +1,6 @@
-# Generated by Django 2.1.5 on 2019-01-21 08:50
+# Generated by Django 2.1.5 on 2019-01-22 10:08
 
+from django.conf import settings
```

Comment:
@calvinpete Seems you have merged migrations. Please make new migrations other than these initial migrations, so that the db can be both downgraded and upgraded.